packages/wizard_router/lib/src/controller.dart

```diff
@@ -26,6 +27,11 @@ class WizardController extends SafeChangeNotifier {
         next = await getRoute(next.name!);
       }
       return next;
+    } on Exception catch (e) {
```

Comment:
We'd probably want a catch-all here? `Exception` doesn't cover everything that can be thrown in Dart

Reply:
After re-reading some docs on `Error` and `Exception` I concluded we shouldn't catch anything more general than `Exceptions` (and ideally try to reduce the number of catches without `on` clauses elsewhere). It suffices to catch `GrpcError` (which is badly named, as it implements `Exception`..).

But maybe we don't need to be that cautious? Do you think we should catch everything here?